docs/release/release-note-0.3.0.md

```diff
@@ -0,0 +1,30 @@
+# Release Note 0.3.0
+
+## ONE-vscode
+
+### OneExplorerView
+
+- Support more commands
+  - [Open with Text Editor] Command
+  - [Open Containing Folder] Command
+  - [Collapse All] Command
+- [Improve Create Config] Command
+- Prevent errors on soft link / virtualenv .cfg file
+- Rename related files automatically on cfg edit
+
+### Toolchain
+
+- Show Installable Toolchains on ToolchainView
```

Comment:
If `Show Installable Toolchains` is menu name,

```
- [Show Installable Toolchains] on ToolchainView
```

if it's not menu name,

```
- Show installable toolchains on ToolchainView
```

Reply:
I think this is not a menu name, @jyoungyun am I right?